lib/ReactViewModels/ViewState.js

```diff
@@ -170,7 +169,7 @@ export default class ViewState {
     }
 
     viewCatalogMember(catalogMember) {
-        if (addedByUser(catalogMember)) {
+        if (catalogMember.isUserSupplied === true) {
```

Comment:
I think the original code was correct. Was it causing problems?

Reply:
yeah i think that was some old changes sneaking in from a messy merge